### PR TITLE
Update qqq.json

### PR DESCRIPTION
**Pull request by krabina**
hope this is correct :-)

In reference to https://github.com/SemanticMediaWiki/SemanticApprovedRevs/issues/8

Should prepare for translatewiki setup

### CODE REVIEW

File: i18n/qqq.json [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-			"Shirayuki",
-			"Андрей"
+			"MWJames",
+			"Kghbln"
 		]
 	},
-	"sdata-desc": "{{desc|name=Semantic Extra Special Properties|url=https://www.mediawiki.org/wiki/Extension:Semantic_Extra_Special_Properties}}",
-	"sdata-name": "{{notranslate}}\nName of the extension as it should be displayed."
+	"semantic-approvedrevs-desc": "{{desc|name=Semantic Approved Revs|url=hhttps://www.mediawiki.org/wiki/Extension:Semantic_Approved_Revs}}",
+	"semantic-approvedrevs-name": "{{notranslate}}\nSemantic Approved Revs"
```
Comment (kghbln) [inline i18n/qqq.json]: A comma is missing here at the end of the line.

Comment (mwjames) [inline i18n/qqq.json]: JSON syntax error! You are missing comma at the end.

File: i18n/qqq.json [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+	"semantic-approvedrevs-property-approved-rev": "Approved revision",
```
Comment (kghbln) [inline i18n/qqq.json]: Add something like "The name of the special property." You can use SESP qqq file as an example here. Same for the other lot.

File: i18n/qqq.json [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
+	"semantic-approvedrevs-property-approved-rev-desc": "\"$1\" identifies the revision ID of a page that has been approved and is provided by the [https://www.semantic-mediawiki.org/wiki/Extension:Semantic_Approved_Revs Semantic Approved Revs] extension.",
```
Comment (kghbln) [inline i18n/qqq.json]: This is identical to the content of "en.json". The content of "en.json" is shown to the translator. The content of "qqq.json" is to provide context for better translation results. Add something like `"This message describes a special property.\n\nParameter:\n* $1 holds the name of the special property" ``You can use SESP qqq file as an example here. Same for the whole of the remaining system messages.

File: i18n/qqq.json (hunk 2 above)
Comment (krabina) [reply i18n/qqq.json]: ah, I see. Thank you for the explanation!

**Comment by kghbln** [comment]
Failure unrelated due to https://github.com/SemanticMediaWiki/SemanticMediaWiki/issues/4093

**Comment by kghbln** [comment]
Thanks for tweaking!

**Pull request merged**